### PR TITLE
Update keyboard shortcuts description

### PR DESCRIPTION
**Pull request by efb4f5ff-1298-471a-8973-3d47447115dc**
I was confused by some descriptions so i (tried) to clarify them

### CODE REVIEW

**Review by PikachuEXE — DISMISSED** [review]

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
Todo: update shortcut added in https://github.com/FreeTubeApp/FreeTube/pull/2221

File: usage/keyboard-shortcuts.md [hunk 1]
```diff
@@ -9,23 +9,26 @@ FreeTube features a variety of keyboard shortcuts that you can use while watchin
...
-| Right Arrow  | Forward 5 seconds / the time-skip interval  |
-| 0-9  | Skip through video by percentage (Ex: 3 key to skip through 30% of a video)  |
+| Up Arrow  | Increase volume |
+| Down Arrow  | Decrease volume |
+| Left Arrow  | Rewind X seconds based on the Rewind Interval and Video Playback Rate Interval (Ex: 5 seconds Rewind Interval and Video Playback Rate Interval 1x, rewinds video 5 seconds) |
+| Right Arrow  | Fast-Forward X seconds based on the Fast-Forward Interval and Video Playback Rate Interval (Ex: 5 seconds Fast-Forward Interval and Video Playback Rate Interval 2x, fast-forwards video 10 seconds) |
+| 0-9  | Skip through video by percentage (Ex: 3 key to skip through 30% of a video) |
 | Alt + D or Command/Ctrl + L  | Focus on the search bar |
+| Play or Pause | Play / Pause video |
+| Previous or Next | Skips to the Previous / Next video |
```
Comment (absidue) [inline usage/keyboard-shortcuts.md:31]:
```suggestion
| Previous or Next | Skips to the Previous / Next video in a playlist |
```

**Review by absidue — CHANGES_REQUESTED** [review]
The previous and next shortcuts are playlist only, I feel like this should be specified

**Review by PikachuEXE — APPROVED** [review]
So many keyboard shortcuts...

**Review by absidue — APPROVED** [review]

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
@ChunkyProgrammer 😬

**Pull request merged**